examples/gno.land/p/demo/ownable/ownable_test.gno

```diff
@@ -6,12 +6,12 @@ import (
 )
 
 var (
-	firstCaller  = std.Address("g1l9aypkr8xfvs82zeux486ddzec88ty69lue9de")
-	secondCaller = std.Address("g127jydsh6cms3lrtdenydxsckh23a8d6emqcvfa")
+	firstCaller  = std.Address("g1wymu47drhr0kuq2098m792lytgtj2nyx77yrsm")
```

Comment:
Did you remove my previous comment?

This change is dangerous as it sets the default `gno test` context address as "firstCaller", which hides a bug with `std.TestSetRealm`.

Reply:
no, i have just pushed new commits and do nothing

Reply:
https://github.com/gnolang/gno/pull/2198/commits/8bd3ac0bd411c9f2adee3bbc466e57ed6f33ac5b